src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
[This link](http://stackoverflow.com/questions/9208931/spellcheck-disable-with-css-not-html) has a way to do it in JavaScript.